css/components/lists.css

```diff
@@ -0,0 +1,61 @@
+.list-container {
+  background-color: #f7fafc;
```

Comment:
Suggestion: variable name can be used for the color if it is used at multiple places.